health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/Hypothesis.js

```
@@ -152,16 +152,16 @@ const Hypothesis = ({ category, assumptions: initialAssumptions, setShowToast, a
+            <LabelFieldPair className="mp-hypothesis-label-field" style={{ alignItems: "center"}} key={index}>
               <div className="assumption-label">
                 <span>
                   {`${t(item)}`}
                   <span className="mandatory-span">*</span>
-                  <span className="icon-wrapper">
-                    <TooltipWrapper content={t(`HYPOTHEISIS_MESSAGE_FOR_${item}`)} children={<InfoBannerIcon fill={"#C84C0E"} />} />
-                  </span>
                 </span>
               </div>
+              <div style={{ alignSelf : "center"}}>
+                <TooltipWrapper content={t(`HYPOTHEISIS_MESSAGE_FOR_${item}`)} children={<InfoBannerIcon fill={'#C84C0E'} />} />
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)